app/models/article.js

```diff
@@ -0,0 +1,38 @@
+/**
```

Comment:
What if we tried to isolate those and removed them? Do you think it's a better option rather than create models for which we're not sure of the use just to pass tests?

I'm open for options.